gen/generator.go

```diff
@@ -467,6 +469,7 @@ func renderTemplate(templatePath, outputPath string, config interface{}) {
 			}
 		}
 	}
+	if config.
```

Comment:
This is a typo I believe..

Reply:
Yea I caught it yesterday but havent removed it yet. Just did!